src/batch-client/index.ts

```diff
@@ -153,6 +155,7 @@ import {
 
 import { CASTING_PREFS } from './constants';
 import { Notifier } from './notifier';
+import { some } from 'lodash';
```

Comment:
```suggestion
import some from 'lodash/some';
```

Reply:
when you use new method from lodash in api-client, you will have to modify rollup.config.js to consider it as external dependency

Reply:
Removed as it is unused. may be added automatically.